pkg/monitor/event.go

```diff
@@ -18,8 +18,10 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var ReMatchFirstQuote = regexp.MustCompile(`"([^"]+)"( in (\d+(\.\d+)?(s|ms)$))?`)
+
 func startEventMonitoring(ctx context.Context, m Recorder, client kubernetes.Interface) {
-	reMatchFirstQuote := regexp.MustCompile(`"([^"]+)"( in (\d+(\.\d+)?(s|ms)$))?`)
+	ReMatchFirstQuote := regexp.MustCompile(`"([^"]+)"( in (\d+(\.\d+)?(s|ms)$))?`)
```

Comment:
Could you document what this is intended to match with an example. Little tough to parse.

Reply:
Looks like the original author was trying to parse something like `"something" in 5.6s`.